authors/apps/articles/migrations/0002_auto_20181003_0716.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.1 on 2018-10-03 07:16
```

Comment:
Lets have one migrations file per feature. Delete all your migrations and migrate again to have just one migration file.

Reply:
Ooh. Nice, I was having doubt on that since there were so many migrations. 
I've now done that.